### PR TITLE
use prone to retrieve better frames

### PR DESCRIPTION
**Pull request by piranha**

### CODE REVIEW

**Comment by martinklepsch** [comment]
Hey @piranha this looks good. Are you still interested in getting this merged? @sethtrain was so kind to add me as a collaborator so I would be able to merge this.

I wonder if it would make sense to add some tests? Or do you think the important bits are already tested in the prone tests?

**Comment by piranha** [comment]
I'm not exactly sure what to test, to be honest. Prone tests itself well, so yeah, not sure if it's necessary.

**Comment by martinklepsch** [comment]
That's totally fine too, was just a thought :) Thanks for updating the PR) 👍

**Comment by martinklepsch** [comment]
This is now released as `1.4.0`: https://github.com/sethtrain/raven-clj#changes 🎉

**Comment by martinklepsch** [comment]
Ugh, just spotted a mistake, will release 1.4.1 shortly.

**Comment by martinklepsch** [comment]
Ok, 1.4.1 released with actual dependency on Prone. Please report any bugs.

File: src/raven_clj/interfaces.clj [hunk 1]
```diff
@@ -17,17 +18,31 @@
...
+     :context_line (nth source 5)
+     :pre_context  (take 5 source)
+     :post_context (drop 6 source)}))
 
 (defn stacktrace [event-map ^Exception e & [app-namespaces]]
-  (assoc event-map
-    :exception [{:stacktrace (make-stacktrace-info (.getStackTrace e) app-namespaces)
-                 :type (str (class e))
-                 :value (.getMessage e)}]))
+  (let [stacks (stacks/normalize-exception (stack/root-cause e))
```
Comment (martinklepsch) [inline src/raven_clj/interfaces.clj]: Where is `root-cause` coming from here?

Comment (martinklepsch) [inline src/raven_clj/interfaces.clj]: is it just `clojure.core/root-cause`?

**Comment by martinklepsch** [comment]
`1.4.2` is released with the missing require statements

File: src/raven_clj/interfaces.clj (hunk 1 above)
Comment (piranha) [inline src/raven_clj/interfaces.clj]: Yes! Sorry. :(

**Pull request merged**